### PR TITLE
Tidy up contributing guide, close to finishing!

### PR DESCRIPTION
**Pull request by Sajo811**
Yay

### CODE REVIEW

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 1]
```diff
@@ -1,51 +1,83 @@
...
 # Let's get started
 
-You will need a GitHub account, make sure you have one and are logged into the GitHub website.
+You will need a [GitHub](https://github.com/join) account, make sure you have one and are logged into the GitHub website.
 
-You'll need git _(command line interface needed, none of the GUIs allows access to all the commands being used, it is a bit tedious typing the commands, however, they become second nature with enough edits.)_ and be able to push and pull from your repo.
+You'll need git *(command line interface needed, none of the GUIs allows access to all the commands being used, it is a bit tedious typing the commands, however, they become second nature with enough edits)* and be able to push and pull from your repo.
 
-    TODO: Explain how to do that ^^ since that requires whole guides, that isn't covered here on this 1st go-around.
+Download Git from here: https://git-scm.com/downloads
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: nah, guide needs to be complete - leave the todo

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 2]
```diff
@@ -1,51 +1,83 @@
...
 
-Right now that you have access to git and can pull from/push to your repos on GitHub let's continue with how to create/update/re-arrange content for the TurtleCoin wiki.
+And we'll quickly go over some basic commands:
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: this covers some things, however, if u set up a fresh system and download git and follow this you will run into issues. the guide needs to cover these things.

Pet peeve of mine is reading guides that assume things.

specifically about setting up username/email config and auth things.
can't go from installing git to pushing commits without those 2 things.

so I'd be inclinde to either write a complete guide from zero to pushing commits or finding an existing on and copy/pasting it wholesale into a new page and linking the source.

Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: oh right i forgot about those :/

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 3]
```diff
@@ -1,51 +1,83 @@
...
-This guides purpose is to break down the barrier that exists to get from "I want to write/contribute docs" to "here is my PR". If anything is not clear or could be made clear, please jump into discord #dev_docs channel and chat about it or even go ahead update the doc.
+This guides purpose is to break down the barrier that exists to get from "I want to write/contribute docs" to "here is my PR". If anything is not clear or could be made clear, please jump into the `#dev_docs` channel in [Discord](http://chat.turtlecoin.lol) and chat about it or even go ahead update the doc.
 
 ### Defintions
 
-repo = git repository  
-wiki repo = the git repository that powers the wiki tab in a GitHub repo  
-PR = pull request  
+repo = git repository  [(example)](https://github.com/turtlecoin/turtlecoin-docs)
+wiki repo = the git repository that powers the wiki tab in a GitHub repo  [(Example - You can only clone it, not view it)](https://github.com/turtlecoin/turtlecoin.wiki.git)
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: these examples are inaccurate...

sure you know what you mean, however telling someone a git repo is a GitHub webpage is not ideal.

Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: does it matter if i add a ss later on which points to both local files and the code tab on github as repos

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 4]
```diff
@@ -1,51 +1,87 @@
...
-You'll need git _(command line interface needed, none of the GUIs allows access to all the commands being used, it is a bit tedious typing the commands, however, they become second nature with enough edits.)_ and be able to push and pull from your repo.
+You'll need git *(command line interface needed, none of the GUIs allows access to all the commands being used, it is a bit tedious typing the commands, however, they become second nature with enough edits)* and be able to push and pull from your repo.
 
-    TODO: Explain how to do that ^^ since that requires whole guides, that isn't covered here on this 1st go-around.
+`TODO: Explain how to do that ^^ since that requires whole guides, that isn't covered here on this 1st go-around.`
 
-Right now that you have access to git and can pull from/push to your repos on GitHub let's continue with how to create/update/re-arrange content for the TurtleCoin wiki.
+Download Git from here: https://git-scm.com/downloads
+
+`either make a new guide about setting it up, like email/password auth or find an existing guide and link to it or take the code if permitted`
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: this is confusing - maybe remove it and add the comment to the ToDo above?

git config username/email (not email/password)
you don't need permission to copy/paste stuff - if they don't want it copied they shouldn't post it on the net... blah blah copyright, credit sources all is good - can't link to things that can disappear tomorrow...

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 3 above)
Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: and it's clarified later that repo  can refer to local git repo or code tab on github

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 4 above)
Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: >don't wait it copied they shouldn't post it on the net
wait what's the MIT license all about then

which our own docs are licensed under

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 3 above)
Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: [this is not outdated, dangu githhub]

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 5]
```diff
@@ -1,51 +1,93 @@
...
+
+Then click "Create pull request" and you're done. Thanks for paying attention to the details!
 
 ### Disclaimer
 
-Nothing in this guide is the "right" way, it is simply a "way" that will work 99% of the time and as someone new to the world of git and GitHub, it is going to cut through an almost vertical learning curve that is git.
+Nothing in this guide is the "right" way, it is simply a "way" that will work 99% of the time and as someone new to the world of git and GitHub, is going to cut through an almost vertical learning curve that is git.
 
-As a developer person who is familiar with git and wanting to help with docs, you might find the commands part useful for refreshing on how to interact with the wiki git repo that is in GitHub from the code in the actual repo.
+As a developer-person who is familiar with git and wanting to help with docs, you might find the commands part `/* is that what you're talking about, soregums? a separate section with all the commands? */` useful for refreshing on how to interact with the wiki git repo that is in GitHub from the code in the actual repo.
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: # The commands/workflow process

It's already here.

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 6]
```diff
@@ -1,51 +1,93 @@
...
+If anything is not clear or could be made clear, please jump into the `#dev_docs` channel in [Discord](http://chat.turtlecoin.lol) and chat about it, or even go ahead update the doc.
 
-### Defintions
+### Definitions
 
-repo = git repository  
-wiki repo = the git repository that powers the wiki tab in a GitHub repo  
-PR = pull request  
+repo = git repository  [(example)](https://github.com/turtlecoin/turtlecoin-docs)  
+wiki repo = the git repository that powers the wiki tab in a GitHub repo  [(Example - You can only clone it, not view it)](https://github.com/turtlecoin/turtlecoin.wiki.git)
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: These examples are getting in the way of the definitions as they are inaccurate.

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 7]
```diff
@@ -1,51 +1,93 @@
...
+These are the commands and the workflow. Further down in the guide, 'why' and explanations and scenarios will be covered.  
+Once understood and when you need a reminder of what to do, this is the section you'll be looking for and wanting a quick fix. So it is covered first.  
+Furthermore, the commands are in order, deviating from the order will give interesting results.  
+Again, this is a how-to guide to get from "I want to contribute" to "check out my PR", rather than a "How-to git" guide.  
+For the latter read [pro git](https://git-scm.com/book).
 
 Set the stage:
+
+    /* @soregums, from @sajo8 in his PR. can u explain this? why is this wrapped in code blocks? */
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: Not clear what you mean :/

Set the stage = set context for what is about to take place.

Only code blocks I see are to highlight the chunk of work the `guide` and `branch`

Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: > Only code blocks I see are to highlight the chunk of work the `guide` and `branch`

wut

https://i.imgur.com/q5ZX08j.png

why is this wrapped in codeblocks with asteriks and pluses? why not normal formatting?

Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: https://i.imgur.com/jlR8d3p.png

Comment (SoreGums) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: It's a **blockquote**, cause I indented it, so just needs to be outdented.

3x back ticks = code block, hence why I couldn't find them when reading your description of _why is this a code block._

Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: oh ok

can i outdent it and add the asteriks

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 8]
```diff
@@ -59,87 +101,150 @@ Set the stage:
...
+
+  * 19.8 Since the PR already exists your commits will show up automatically so no need for step **17**
+  * 19.9 You can go back to your previous branch and continue with that work, part of the above 19.7 was going back to master, so this should work fine  
+       `git checkout how-to-contribute-to-wiki`
+  * 19.10 Continue where you were before
+
+  ```
+  */ nice job soregums
+  please address my questions, I wrapped them in code blocks and start/ended with /* and */ (like this)
+  I got a little lost toward the end but I think that's just me
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: Yeah, it's not worded clearly.

This happened, I was working on this guide, then had to make updates to the rain Borg guide.

Will figure out the wording of this and update

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 9]
```diff
@@ -59,87 +101,150 @@ Set the stage:
...
+   * 19.5 Continue from step **7**.
+   * 19.6 Then when you get to step **8** do this instead
+
+        `git push --force wiki HEAD:master`
+
+   * 19.7 Now keep going through the steps until step **16**, going to do it a bit differently
+
+        `lol - need to figure this out`  
+        ```
+        /* idk but I think u mention above that the branch name can be whatever and u follow 16 but merge the branch of the name which u made? */
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: It's different.
The refs change so it is not the same steps.

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 10]
```diff
@@ -59,87 +101,150 @@ Set the stage:
...
+
+     ```
+     git add .
+     git commit -m "your message here"
+     git status
+     ```
+     It should all be clean now
+
+  19.4 Now that your current work is saved, switch back to master and checkout the previous branch  
+`/*it says "switch back to master", but you move to the new branch? also, how did this name come up?*/`
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: Yeah, I missed steps here in the cmds to run.

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md [hunk 11]
```diff
@@ -59,87 +101,150 @@ Set the stage:
...
+
+
+10. Pull the wiki repo back down locally
+   * `git pull wiki`
+
+
+11. Make changes again locally
+   * Edit the file, maybe add an image?
+   * `git add .` will add all files that have been changed
+   * `git commit -m "your-message-here"`
```
Comment (SoreGums) [inline guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: Git commit your message here isn't what I want in here on purpose.

Especially for the squash one.
People need to learn up front about summary and details and don the commit msg nicely from the start.
If a dev already they know how to -m message here

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 7 above)
Comment (SoreGums) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: Yup

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 8 above)
Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: k
ill leave that comment in for now

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 11 above)
Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: you can't `git commit` you *need* a message

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 9 above)
Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: k

File: guides/How-to-contribute-to-the-TurtleCoin-wiki.md (hunk 11 above)
Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: you end up needing to write a message right?

then you have to tell em to press escape then that :qr or :rq or whatever it was

Comment (SoreGums) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: Yes, that's why there is supposed to be an explanations and scenarios section.

You're not wrong, all that is happening is your goal for this is not the same as mine :)

Comment (Sajo811) [reply guides/How-to-contribute-to-the-TurtleCoin-wiki.md]: fair enough 🤷‍♂️

**Comment by Sajo811** [comment]
doing from new acc

**Comment by Sajo8** [comment]
@SoreGums we need to finish this lad

**Pull request closed without merging**